### DISCUSSION TITLE
SpeedyBee35 - the OSD is not showing after switch to iNAV

### DISCUSSION THREAD

**Discussion by AerialSimonH78** [iNavFlight/inav#10383] [General]
I have a SpeedyBee35 it was flying on Betaflight all ok - but have just switched over the iNAV 7.1.2 on it all setup and works apart from I have no OSD showing! - Its analog works perfectly just flashed the iNAV and went through the settings - what should I check or missed??

**Comment by sensei-hacker** [comment]
First thing to check - did you configure and turn on the OSD?

**Comment by daijoubu** [comment]
Did you upload the Inav font?

**Comment by CorvusMonitoring** [comment]
I have the same problem here.
Concerning the Inav font, it seems that they are not accepted by the board: When I upload them, then reboot, the configartion panel shows old fonts again, as if I did not changed them.

**Reply by MrD-RC** [reply]
The font upload in configurator is for analogue only. 

Please make sure you have set up the HD correctly. It is *not* the same as BetaFlight. https://youtu.be/elq6AF5gdXE

**Comment by CorvusMonitoring** [comment]
Oooh yes, i set it up correctly, following dozen of tutorials and advices. 
it is not a config issue…

**Comment by CorvusMonitoring** [comment]
Can I precise that there is already a bug about that board, as the UART2 does not appear on the ports tab.
I switched to different ports on the board, no results, and different power supply (as suggested n another topic), no results.
